orchagent/portsorch.cpp

```diff
@@ -3689,6 +3753,12 @@ bool PortsOrch::removeVlan(Port vlan)
        return false;
     }
 
+    if (!removeVlanHostIntf(vlan))
```

Comment:
This must be called only if `host_intf_id ` is present, else skip

Reply:
done